nucypher/characters.py

```diff
@@ -207,9 +207,13 @@ def start_learning(self):
     def handle_learning_errors(self, *args, **kwargs):
         self.log.warning("Unhandled error during node learning: {}".format(args))
 
-    def select_teacher_nodes(self):
+    def shuffled_known_nodes(self):
         nodes_we_know_about = list(self._known_nodes.values())
```

Comment:
+1 Derives a new list for use as teacher nodes.

Reply:
:heavy_check_mark: